src/applications/lgy/coe/containers/IntroductionPage.jsx

```diff
@@ -23,14 +24,20 @@ function IntroductionPage(props) {
     content = notLoggedInContent(props);
   }
   if (props.loggedIn && coeCallEnded.includes(props.status)) {
-    content = loggedInContent();
+    content = (
+      <div>
+        <COEIntroPageBox coe={props.coe} status={props.status} />
+        {props.coe.status !== 'denied' && <LoggedInContent />}
```

Comment:
It might be worthwhile to import the `COE_ELIGIBILITY_STATUS` constant and use it in this check: `props.coe.status !== COE_ELIGIBILITY_STATUS.denied`

Reply:
Good idea. Im gonna import this into the `<COEIntroPageBox />` component too for the switch statement